backends/ebpf/kernel.mk

```diff
@@ -0,0 +1,91 @@
+# Argument for the CLANG compiler
+LLC ?= llc
+CLANG ?= clang
+INCLUDES+= -I./
+LIBS+=
+# Optimization flags to save space
+CFLAGS+= -O2 -g -D__KERNEL__ -D__ASM_SYSREG_H -Wno-unused-value -Wno-pointer-sign \
+		-Wno-compare-distinct-pointer-types \
+		-Wno-gnu-variable-sized-type-not-at-end \
+		-Wno-address-of-packed-member -Wno-tautological-compare \
+		-Wno-unknown-warning-option
+
+# Arguments for the P4 Compiler
```

Comment:
Add a TODO here, we may need to use the installation folder for p4include, and not ./